### PR TITLE
remove existing APIs from the missing API list

### PR DESCRIPTION
**Pull request by mairaw**
APIs have been onboarded and redirection is bad at the moment

Dependent on PR #10858

### CODE REVIEW

File: _zip/missingapi.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ references:
   name.vb: ToString
   fullName: XDocument.ToString
   fullName.vb: XDocument.ToString
-- uid: System​.Runtime​.CompilerServices.IndexerNameAttribute
+- uid: System.Runtime.CompilerServices.IndexerNameAttribute
```
Comment (rpetrusha) [inline _zip/missingapi.yml]: This isn't a missing API: https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.indexernameattribute

**Review by rpetrusha — APPROVED** [review]
LGTM, @mairaw. I left one comment for you to consider.

**Comment by mairaw** [comment]
Thanks @rpetrusha. Your comment made me realize we could probably remove much more APIs.

Kept the ones that are still missing and opened an mdoc issue:
https://github.com/mono/api-doc-tools/issues/426

File: _zip/missingapi.yml (hunk 1 above)
Comment (kanti3) [reply _zip/missingapi.yml]: #10829
Advanced (Windows Presentation Foundation)
2 minutes to read • Edit Online
In ThisSection
This section describes some of theadvanced areas in WPF.
WPF Architecture
XAML in WPF
BaseElement Classes
ElementTreeand Serialization
WPF Property System
Events in WPF
Input
Drag and Drop
Resources
Documents
Globalization and Localization
Layout
Migration and Interoperability
Performance
Threading Model
Unmanaged WPF API Reference

**Pull request merged**